packages/core/EventManager.ts

```diff
@@ -435,14 +437,14 @@ export default class EventManager {
   private async updateAllCalendarEvents(
     event: CalendarEvent,
     booking: PartialBooking
-  ): Promise<Array<EventResult<NewCalendarEventType>>> {
+  ): Promise<Array<EventResult<NewCalendarEventType>> | []> {
```

Comment:
```suggestion
  ): Promise<Array<EventResult<NewCalendarEventType>>> {
```

No need to update the signature. An empty array is a valid array for the current signature.